__tests__/app.test.js

```diff
@@ -49,20 +54,6 @@ describe("GET /api/articles/:article_id returns correct article", () => {
       },
     });
   });
-  test("endpoint returns correct article", async () => {
-    const results = await request(app).get("/api/articles/3").expect(200);
-    expect(results.body).toEqual({
-      article: {
-        article_id: 3,
-        author: "icellusedkars",
-        body: "some gifs",
-        created_at: "2020-11-03T09:12:00.000Z",
-        title: "Eight pug gifs that remind me of mitch",
-        topic: "mitch",
-        votes: 0,
-      },
-    });
-  });
```

Comment:
Is there a reason this test was deleted? It looks pretty good!

Reply:
Rose told me too lool she says it looked too similar to the next one! It ran just fine however